flink-table/pom.xml

```diff
@@ -45,6 +45,7 @@ under the License.
 		<module>flink-table-uber-blink</module>
 		<module>flink-sql-client</module>
 		<module>flink-sql-parser</module>
+		<module>flink-sql-parser-hive</module>
```

Comment:
must we create a new module?

Reply:
A separate module makes things a little cleaner. Otherwise the pom file of `sql-parser` will become messy and it can get even worse if we add more dialects in the future.